### PR TITLE
clean up registration and registry creation flow to make more consistent

### PR DESCRIPTION
**Pull request by jake-kent**

### CODE REVIEW

File: Maco/accounts/forms.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from django.utils.translation import ugettext_lazy as _
 
 from Maco.wishlists.constants import ADDRESS_TYPES, CLIMATES  # NOQA
+from Maco.wishlists.models import WishList  # NOQA
```
Comment (gone) [inline Maco/accounts/forms.py]: why no qa this

File: Maco/accounts/views.py [hunk 2]
```diff
@@ -96,6 +95,11 @@ def get_initial(self):
 
     def form_valid(self, form):
         form.save()
+        if 'viewregistry' in form.data:
```
Comment (gone) [inline Maco/accounts/views.py]: Rather than assign as a property, just use get_success_url()

File: Maco/home/tests.py [hunk 3]
```diff
@@ -13,3 +18,37 @@ def test_error_route(self):
         visit = lambda: self.client.get(reverse('error'))
         self.assertRaises(Exception, error)
         self.assertRaises(Exception, visit)
+
+def create_wishlist(user):
```
Comment (gone) [inline Maco/home/tests.py]: The fact this is used in multiple files makes me think we should make a user model method to do this

File: Maco/accounts/forms.py (hunk 1 above)
Comment (jake-kent) [reply Maco/accounts/forms.py]: for some reason on all of our recent projects importing from {project}.{module}.{module} is disliked by the linter

**Pull request merged**